openapi/src/main/java/io/helidon/openapi/OpenAPISupport.java

```diff
@@ -620,6 +623,27 @@ public OpenAPISupport build() {
             return new OpenAPISupport(this);
         }
 
+        /**
+         * Set various builder attributes from the specified {@code Config} object.
+         * <p>
+         * The {@code Config} object can specify {@value #CONFIG_PREFIX}.web-context
+         * and {@value #CONFIG_PREFIX}.static-file in addition to settings
+         * supported by {@link OpenAPIConfigImpl.Builder}.
+         *
+         * @param config the {@code Config} object possibly containing settings
+         * @exception NullPointerException if the provided {@code Config} is null
+         * @return updated builder instance
+         */
+        public Builder helidonConfig(Config config) {
+            config.get(CONFIG_PREFIX + ".web-context")
```

Comment:
You should not use the  `CONFIG_PREFIX` - the config should be on the node of `openapi`. Microprofile component configuring this class should request that node before calling this method.

The method should be called `config` - that is the rule for any builder in any Helidon component.